community/kernel/src/main/java/org/neo4j/kernel/impl/util/dbstructure/InvocationTracer.java

```diff
@@ -131,7 +131,7 @@ private void formatPreamble( Appendable builder ) throws IOException
         }
         formatln( builder );
         formatln( builder, "//" );
-        formatln( builder, "// GENERATED FILE. DO NOT EDIT. " );
+        formatln( builder, "// GENERATED FILE. DO NOT EDIT." );
```

Comment:
That was deliberately included because it turns out that the generated code fails the checkstyle. Even though we manually edited the generated file and fixed this manually, we thought it correct to include this fix in the generator so that the next person to use this capability would not get burned as well. While we could have made a separate PR for this, a whole PR for removal of one space character seems silly, and this problem was related to our work, so this seemed best.